doc/releases/changelog-0.27.0.md

```diff
@@ -250,7 +250,7 @@
   ```pycon
   >>> H2datasets = qml.data.load("qchem", molname="H2", basis="STO-3G", bondlength=1.1)
   >>> print(H2datasets)
-  [<pennylane.data.dataset.Dataset object at 0x7f14e4369640>]
+  [<Dataset = description: qchem/H2/STO-3G/1.1, attributes: ['molecule', 'hamiltonian', ...]>]
   >>> H2data = H2datasets[0]
```

Comment:
Could you ensure the `H2data` thing is consistent in the changelog?

Reply:
Yup, currently lines 330-332 use `H2_dataset` instead of `H2data`

Reply:
updated